src/ffi.rs

```diff
@@ -5,6 +5,7 @@ extern crate libc;
 pub type rill_key_t = u64;
 pub type rill_val_t = u64;
 pub type rill_ts_t = u64;
+pub type rill_col = i32; /* enums are 32bit */
```

Comment:
Consider using an actual enum as you can use the `#[repr(C)]` to make sure it matches the representation. I would also create another enum in the `rill` namespace so that the user doesn't have to write `rill_col_a` and can just use `rill::col_a` instead.